ad2web/notifications/types.py

```diff
@@ -1175,6 +1175,8 @@ def _do_get(self, data, app=None):
     @raise_with_stack
     def send(self, type, text, raw):
         self.msg_to_send = text
+        message_string = 'Primary Residence Alarm Event: %s' % text
```

Comment:
The user who contributed this change in #99 stated that this string is hardcoded for his particular use case:

> the new "message_string" variable includes a version of the "Subject" option from the email notification, but I just harcoded in the message instead of modifying the web forms to offer it as a front-end-selectable option, mainly because I'm lazy. Presumably, if you're applying this patch, you can just type in whatever you want there or just set it equal to the 'text' variable to eliminate it entirely.

Therefore, it doesn't make much sense to merge this change.

I would recommend either amending this change to get text from the web form or removing it from this PR altogether.